### PR TITLE
Latest flight services completed.

### PR DESCRIPTION
**Pull request by AlejandroPiedrahita**
The last three missing services of the flight api have been completed.

### CODE REVIEW

File: travel-agency/reservation-management/src/main/resources/application.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 server:
-  port: 0
+  port: 8084
```
Comment (JuanCa2002) [inline travel-agency/reservation-management/src/main/resources/application.yml]: El puerto jeje

File: travel-agency/reservation-management/src/main/kotlin/com/co/unitravel/infrastructure/config/ReservationOpenAPIConfig.kt [hunk 2]
```diff
@@ -6,7 +6,7 @@ import io.swagger.v3.oas.models.servers.Server
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
-const val SERVER_PORT = "2208"
+const val SERVER_PORT = "8084"
```
Comment (JuanCa2002) [inline travel-agency/reservation-management/src/main/kotlin/com/co/unitravel/infrastructure/config/ReservationOpenAPIConfig.kt]: El puerto x2

**Review by JuanCa2002 — CHANGES_REQUESTED** [review]

File: travel-agency/reservation-management/src/main/resources/application.yml (hunk 1 above)
Comment (AlejandroPiedrahita) [reply travel-agency/reservation-management/src/main/resources/application.yml]: xD

File: travel-agency/reservation-management/src/main/kotlin/com/co/unitravel/infrastructure/config/ReservationOpenAPIConfig.kt (hunk 2 above)
Comment (AlejandroPiedrahita) [reply travel-agency/reservation-management/src/main/kotlin/com/co/unitravel/infrastructure/config/ReservationOpenAPIConfig.kt]: xD x2

**Review by JuanCa2002 — APPROVED** [review]

**Pull request merged**